tests/components/letpot/test_init.py

```diff
@@ -0,0 +1,98 @@
+"""Test the LetPot integration initialization and setup."""
+
+from unittest.mock import MagicMock
+
+from letpot.exceptions import LetPotAuthenticationException, LetPotConnectionException
+import pytest
+
+from homeassistant.components.letpot.const import DOMAIN
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+
+from . import setup_integration
+
+from tests.common import MockConfigEntry
+
+
+@pytest.mark.freeze_time("2025-01-31 00:00:00")
+async def test_load_unload_config_entry(
+    hass: HomeAssistant,
+    mock_config_entry: MockConfigEntry,
+    mock_client: MagicMock,
+    mock_device_client: MagicMock,
+) -> None:
+    """Test config entry loading/unloading."""
+    await setup_integration(hass, mock_config_entry)
+
+    assert mock_config_entry.state is ConfigEntryState.LOADED
+    assert len(mock_client.refresh_token.mock_calls) == 0  # Didn't refresh valid token
```

Comment:
```suggestion
    mock_client.refresh_token.assert_not_called()  # Didn't refresh valid token
```
I believe this also a thing, also `assert_called_once()`